pynitrokey/tlv.py

```diff
@@ -0,0 +1,81 @@
+from typing import Optional, Sequence, Tuple
```

Comment:
Just being curious:  How does this module relate to our tlv8 dependency?

Reply:
Maybe I should migrate the secrets app tests to use this TLV implementation too.

Reply:
Oh, its also used in the sdk I missed that.

Reply:
Wait, tlv8 is not actually the full implementation of the BER-TLV standard, but they appear to have a compatible substet which secrets app uses. Anyway we can't use it here, PIV uses tags that are greater than 255.